clients/src/main/java/org/apache/kafka/common/requests/LeaveGroupResponse.java

```diff
@@ -41,20 +43,33 @@ public LeaveGroupResponse(Struct struct, short version) {
         this.data = new LeaveGroupResponseData(struct, version);
     }
 
-    public LeaveGroupResponseData data() {
-        return data;
-    }
-
     @Override
     public int throttleTimeMs() {
         return data.throttleTimeMs();
     }
 
-    public Errors error() {
-        return Errors.forCode(data.errorCode());
+    /**
+     * Get leave group errors.
+     *
+     * Possible error code:
```

Comment:
I think `FENCED_INSTANCE_ID` is also possible?

Reply:
If we add this handling to the group coordinator, yes